tracker/js_tracking_api.rst

```diff
@@ -410,6 +410,33 @@ Tracking Custom Dimension
 `````````````````````````
 .. todo:: Maybe add section about using tag template from Tag Manager.
 
+If you wish to set a custom dimension to use it in tracking functions, use the ``setCustomDimensionValue`` function::
+
+    _paq.push(["setCustomDimensionValue", customDimensionID, customDimensionValue]);
+
+.. describe:: customDimensionID
+
+    **Required** ``number`` ID of dimension
+
+.. describe:: customDimensionValue
+
+    **Required** ``string`` Value of Custom Dimension - limited to 255 characters.
+
+.. warning::
+
+    When you set a Custom Dimension, that value will be used in all tracking requests within a page load.
+
+.. warning::
+    This function does not send any data to the :term:`Analytics`. It sets a Custom Dimension to be sent with following events (e.g. page view, ecommerce events, outlink or download events).
+
+
+Example of usage::
+
+    _paq.push(["setCustomDimension", 3, "loginStatus"]);
```

Comment:
I think we should remove uses of deprecated methods from examples. This applies to the whole document.
```suggestion
    _paq.push(["setCustomDimensionValue", 3, "loginStatus"]);
```

Reply:
I updated example for `setCustomDimensionValue` function, but I'm not sure we can remove other examples. The "JavaScript tracking API" page is mostly covered by description of command and examples of use since these are not standard JS functions and other example of use describes deprecated `setCustomDimension` command.

Reply:
So you're saying that this page is not about `_paq.push(["setCustomDimensionValue"...` but rather javascript methods and that examples with deprecated `setCustomDimension` are fine because they are used to show analogies. Is that correct?

In that case I see two options:
1.Keep using `setCustomDimension` and only that one for consistency but mention that `setCustomDimensionValue` should generally be used instead.
2. Change all examples to `setCustomDimensionValue`

Either is fine for me, what matters is that people don't get confused because we're mixing both methods.